examples/AllTests/CMakeLists.txt

```diff
@@ -10,9 +10,14 @@ add_executable(ExampleTests
 
 target_include_directories(ExampleTests
     PRIVATE
-        ../ApplicationLib
+        .
 )
 
-cpputest_normalize_test_output_location(ExampleTests)
-target_link_libraries(ExampleTests ApplicationLib ${CppUTestLibName} ${CppUTestExtLibName})
+target_link_libraries(ExampleTests
+    PRIVATE
+        ApplicationLib
+        CppUTest::CppUTestExt
```

Comment:
Note that the transitive dependency (ExampleTests -> CppUTestExt -> CppUTest) works even using `find_package`.